vtex-apps/b2b-organizations
# [B2BORG-120] Check duplicate address in cost center

#### What problem is this solving?

Added addressID duplicate check, and set hashcode of the address string as the addressID

#### How to test it?

1. Go to the [Workspace](https://aurora1--b2bstoreqa.myvtex.com/) and login
(note: create a cost center if you don't have one already)
2. Add a new address in your cost center
3. Add another new address in your cost center that is identical to the address you added in step 2
4. You should see a toast message that won't allow you to add the duplicate address

Please test for both the storefront and the admin.
